src/kontrol/foundry.py

```diff
@@ -835,3 +825,12 @@ def foundry_node_printer(foundry: Foundry, contract_name: str, proof: APRProof)
     if type(proof) is APRProof:
         return FoundryAPRNodePrinter(foundry, contract_name, proof)
     raise ValueError(f'Cannot build NodePrinter for proof type: {type(proof)}')
+
+
+def load_foundry(foundry_root: Path, bug_report: BugReport | None = None) -> Foundry:
```

Comment:
I think this belong to `__main__`, printing to `stderr` and exiting with an error code is only for command line use.

Reply:
Moving this in `__main__` would prevent me from importing the function in the test files.

Reply:
But I guess this function is not needed in there.